src/PrestaShopBundle/Resources/config/routing/admin/configure/shop_parameters/search_preferences.yml

```diff
@@ -0,0 +1,28 @@
+# @TODO uncomment legacy links when page is done
+admin_search_index:
+  path: /
+  methods: [ GET ]
+  defaults:
+    _controller: 'PrestaShopBundle\Controller\Admin\Configure\ShopParameters\SearchController::indexAction'
+    _legacy_controller: AdminSearchConf
+    # _legacy_link: AdminSearchConf
+    # _legacy_feature_flag: search
+
+admin_contacts_search:
+  path: /
+  methods: [ POST ]
+  defaults:
+    _controller: PrestaShopBundle\Controller\Admin\CommonController::searchGridAction
+    gridDefinitionFactoryServiceId: prestashop.core.grid.definition.factory.contacts
```

Comment:
contacts?
Check other places too, probably left some copy paste mistakes (I assume you copied from contacts grid?)